core/src/main/java/kafka/autobalancer/metricsreporter/AutoBalancerMetricsReporter.java

```diff
@@ -213,8 +214,18 @@ public void configure(Map<String, ?> configs) {
         if (brokerRack == null) {
             brokerRack = "";
         }
-        brokerNwInCapacity = reporterConfig.getDouble(AutoBalancerMetricsReporterConfig.AUTO_BALANCER_BROKER_NW_IN_CAPACITY);
-        brokerNwOutCapacity = reporterConfig.getDouble(AutoBalancerMetricsReporterConfig.AUTO_BALANCER_BROKER_NW_OUT_CAPACITY);
+
+        Object s3BaseLine = configs.get(S3_NETWORK_BASELINE_BANDWIDTH);
+
+        Object nwInCapacity = configs.get(AutoBalancerMetricsReporterConfig.AUTO_BALANCER_BROKER_NW_IN_CAPACITY);
```

Comment:
maybe it's better to do something like:
if s3Basline != null and !configs.contains(autobalancerNw...):
    then configs.put(autobalancerNw..., s3Basline)
...
then use modified $configs to construct AutoBalancerMetricsReporterConfig, and then just get autobalancer nw config from reporterConfig as usual